test/task-helpers.spec.coffee

```diff
@@ -25,3 +25,8 @@ describe 'TaskHelpers', ->
     )
     expect( lateness.is_late ).to.be.true
     expect( lateness.how_late ).to.equal('30 minutes')
+
+  it 'converts chapter_sections to numbers', ->
+    expect(Helpers.chapterSectionToNumber([1])).to.equal(100)
+    expect(Helpers.chapterSectionToNumber([1, 2])).to.equal(102)
+    expect(Helpers.chapterSectionToNumber([31, 88, 42])).to.equal(318842)
```

Comment:
Probably not.  The algorithm doesn't do anything special to handle them but the test doesn't need to explicitly test them